aspnetcore/security/authentication/identity-enable-qrcodes.md

```diff
@@ -61,10 +73,16 @@ private string GenerateQrCodeUri(string email, string unformattedKey)
 }
 ```
 
-The second parameter in the call to `string.Format()` is your site name, taken from your solution name. It can be changed to anything value, but it must always be URL encoded.
+The second parameter in the call to `string.Format` is your site name, taken from your solution name. It can be changed to any value, but it must always be URL encoded.
 
 ## Using a different QR Code library
 
-You may have a preferred QR Code library which you want to use. The `qrCode` element in the view is a suitable place to put the QR code.
+You can replace the QR Code library in the `qrCode` element.  <!-- Original
```

Comment:
@blowdart  either way. You can use the edit to directly fix it before the merge.